ocs2_robotic_examples/ocs2_legged_robot_ros/src/LeggedRobotPoseCommandNode.cpp

```diff
@@ -123,10 +124,21 @@ int main(int argc, char* argv[]) {
 
   // goalPose: [deltaX, deltaY, deltaZ, deltaYaw]
   const scalar_array_t relativeBaseLimit{10.0, 10.0, 0.2, 360.0};
-  TargetTrajectoriesKeyboardPublisher targetPoseCommand(nodeHandle, robotName, relativeBaseLimit, &commandLineToTargetTrajectories);
 
   const std::string commandMsg = "Enter XYZ and Yaw (deg) displacements for the TORSO, separated by spaces";
-  targetPoseCommand.publishKeyboardCommand(commandMsg);
+  
+  bool use_joystick;
+  nodeHandle->get_parameter("use_joystick", use_joystick);
```

Comment:
@kylc this was my attempt at reading a parameter to either use the keyboard or joystick, but its not working unfortunetaly, I think since the node is not spinning all the time and when its blocked to keyboard command then its not able to listen to parameter change?

Reply:
It doesn't have to be a ROS parameter. You could load a parameter from the OCS2 config file, like they do just a couple of lines above this.

https://github.com/PickNikRobotics/ocs2/blob/e4aaaa696a0b851094e2bb461a769b661d2cbf1e/ocs2_robotic_examples/ocs2_legged_robot_ros/src/LeggedRobotPoseCommandNode.cpp#L123